archivist_samples/wipp/README.md

```diff
@@ -0,0 +1,203 @@
+# Waste Isolation Pilot Plant (WIPP) Sample
+
+Tracking the Nuclear Waste Management lifecycle is an important aspect to ensure that all safety protocols and procedures have been successfully executed.  Fragmented communication and manual checks can lead to honest mistakes and redundancy.  Digitizing the lifecycle and exposing data to the right parties at the right time can decrease honest mistakes and increase effective communication.  
+
+RKVST Continuous Assurance Hub offers a solution to fragmented communication and manual checks. Parties have near real-time access to data increasing seamless and effective communication and stakeholders can control the sharing of data ensuring one can view information that is relevant.  Policies, procedures and images can be included/attached thus reducing multiple checks and 
+providing persons with the most recent documentation.
+ 
+This sample shows how to quickly get started with integrating Nuclear Waste Management lifecycle with RKVST Continuous Assurance Hub.
```

Comment:
"This sample uses publicly-available information about WIPP (the Waste Isolation Pilot Plant) how to quickly get started with integrating Nuclear Waste Management lifecycle with RKVST Continuous Assurance Hub."

Reply:
Updated